src/StoreKeeper/WooCommerce/B2C/Imports/AttributeImport.php

```diff
@@ -118,5 +118,6 @@ protected function processItem($dotObject, array $options = [])
         }
 
         WooCommerceAttributeMetadata::setMetadata($attribute_id, 'storekeeper_id', $dotObject->get('id'));
+        WooCommerceAttributeMetadata::setMetadata($attribute_id, 'weight', $dotObject->get('order'));
```

Comment:
any reason why is it called `weight` instead of order?

Reply:
Got used to using `weight` for ordering. Changed it to `attribute_order`.